lib/attrsets.nix

```diff
@@ -278,6 +278,23 @@ rec {
     # The set to get the named attributes from
     attrs: genAttrs names (name: attrs.${name});
 
+  /*
+    Like `getAttrs`, but skip attributes not in the input attribute set
+    instead of throwing the "attribute is missing" error.
+
+    Example:
+      getExistingAttrs [ "a" "b" "d" ] { a = 1; b = 2; c = 3; }
+      => { a = 1; b = 2; }
+
+    Type:
+      getAttrs :: [String] -> AttrSet -> AttrSet
```

Comment:
It appears to have reverted to the original text

Reply:
Oops! Thanks for pointing out.

Addressed.